packages/machine/test/integration/test-response-sink.ts

```diff
@@ -24,15 +23,13 @@ export class TestResponseSink implements cf.legacy.node.ResponseSink {
   public store: TestCommitmentStore;
   public signingKey: ethers.utils.SigningKey;
 
-  private requests: Map<string, Function>;
-  private messageListener?: Function;
+  private runProtocolContinuation?: Function;
```

Comment:
Can the typing here be tightened at all for input/output accounting instead of a generic `Function`?

Reply:
Looks like it only ever accepts `cf.legacy.node.WalletResponse`